README.md

```diff
@@ -1,5 +1,4 @@
 [![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
-[![Python 3.8](https://img.shields.io/badge/python-3.8-blue.svg)](https://www.python.org/downloads/release/python-380/)
```

Comment:
thanks for the PR, Ben. Any particular reason to remove the 3.8 badge?

Reply:
I think it works with 3.8, but are we currently testing against it?